### ISSUE TITLE
Place the live search box on the bottom of the selectpicker dropdown.

### ISSUE THREAD

**Issue by acrolink** [silviomoreto/bootstrap-select#1672]
That would be a great featue I quess since the search input box would appear near the field. Thank you.

**Comment by caseyjhol** [comment]
Are you talking about if the menu opens above instead of below?

**Comment by acrolink** [comment]
Exactly. As for me, I have already forked the repository and made a tiny code modification in order to achieve the desired result. I imagine other users might be interested in having an option which allows placement selection.

**Comment by jonasgroendahl** [comment]
This would be great!

**Comment by NicolasCARPi** [comment]
@acrolink why not make a PR and share your modifications if you think others would benefit from it?
